app/components/edit/ProvidedService.tsx

```diff
@@ -50,6 +50,15 @@ export interface InternalSchedule {
   Sunday: InternalScheduleDay[];
 }
 
+/** If the argument is either an InternalSchedule or an empty object, return
+ *  true if it is an InternalSchedule.
+ */
+export function isNonEmptyInternalSchedule(
```

Comment:
Very small nit but it might be more clear to name this function `isEmptyInternalSchedule` and to check if "Monday" is not in the schedule?

Reply:
Yep, good idea, flipped it in 817f0d76dab5f1482f55c20e4f00fe2fa99be72e.